demos/fleet_provisioning/fleet_provisioning_with_csr/pkcs11_operations.c

```diff
@@ -582,7 +601,7 @@ static CK_RV provisionPrivateRSAKey( CK_SESSION_HANDLE session,
         {
             { CKA_CLASS,            NULL /* &privateKeyClass */, sizeof( CK_OBJECT_CLASS )                     },
             { CKA_KEY_TYPE,         NULL /* &privateKeyType */,  sizeof( CK_KEY_TYPE )                         },
-            { CKA_LABEL,            label,                       ( CK_ULONG ) strlen( ( const char * ) label ) },
+            { CKA_LABEL,            ( void * ) label,            ( CK_ULONG ) strlen( ( const char * ) label ) },
```

Comment:
Same comment, the `const char *`  typecast can be removed

Reply:
Its needed; same as above.